pkg/nrtupdater/nrtupdater.go

```diff
@@ -126,7 +126,10 @@ func (te *NRTUpdater) Run(infoChannel <-chan MonitorInfo) chan<- struct{} {
 				}
 				tsEnd := time.Now()
 
-				log.Printf("update request received at %v completed in %v", tsBegin, tsEnd.Sub(tsBegin))
+				tsDiff := tsEnd.Sub(tsBegin)
+				prometheus.OperationDelay.Set(float64(tsDiff.Milliseconds()))
+				log.Printf("update request received at %v completed in %v", tsBegin, tsDiff)
```

Comment:
please remove this log

Reply:
we need to record the delay also in
```func (rm *ResourceMonitor) Run(eventsChan <-chan PollTrigger) (<-chan nrtupdater.MonitorInfo, chan<- struct{}) {```
in
`pkg/resourcetopologyexporter/resourcetopologyexporter.go`
We will need to use labels to differentiate the two values.